server/core/storage.go

```diff
@@ -159,6 +160,16 @@ func (s *Storage) LoadRegions(f func(region *RegionInfo) []*RegionInfo) error {
 	return loadRegions(s.Base, f)
 }
 
+// LoadRegionsOnce loads all regions from storage to RegionsInfo.Only load one time from regionStorage.
+func (s *Storage) LoadRegionsOnce(f func(region *RegionInfo) []*RegionInfo) error {
+	if atomic.CompareAndSwapInt32(&s.regionLoaded, 0, 1) {
```

Comment:
if we do not `useRegionStorage`, the region does not sync by `region_syncer`, that means the BasicCluster do not have the regions, we need to load it from kv(etcd).

Reply:
Yes,It's a question.When the follower became leader firstly,It's OK.But the follower became leader secondly, the BasicCluster do not load regions from kv(etcd)